### PR TITLE
U4X-634: Create Attachment concepts to resolve issue of it being created auto by attachment module

### PR DESCRIPTION
**Pull request by slubwama**
https://metsprogramme.atlassian.net/browse/U4X-634

### CODE REVIEW

File: api/src/main/resources/metadata/concepts_and_drugs/Concept.xml [hunk 1]
```diff
@@ -4877,4 +4877,7 @@
     <concept  concept_id="175393"  retired="0" datatype_id="4"  class_id="5"  is_set="0"  creator="1"  date_created="2024-05-09 13:41:10"  version=""  changed_by="1"  date_changed="2024-05-09 13:41:10" uuid="c07bd21c-8a0d-4812-9887-3308ee7f5e42"/>
     <concept  concept_id="175392"  retired="0" datatype_id="2"  class_id="1"  is_set="0"  creator="1"  date_created="2024-05-09 12:45:52"  version=""  changed_by="1"  date_changed="2024-05-09 12:45:52" uuid="b2e6884d-2354-4d93-ab22-ec9d2b262177"/>
     <concept  concept_id="175391"  retired="0" datatype_id="2"  class_id="1"  is_set="0"  creator="1"  date_created="2024-05-09 12:40:41"  version=""  changed_by="1"  date_changed="2024-05-09 13:48:23" uuid="96370f02-aab1-4ec2-a42e-df73f63f12d4"/>
+    <concept  concept_id="186350"  retired="0" datatype_id="4"  class_id="9"  is_set="1"  creator="1"  date_created="2024-06-18 12:40:41"  version=""  changed_by="1"    uuid="96370f02-aab1-4ec2-a42e-df73f63f12d4"/>
```
Comment (Mwanje) [inline api/src/main/resources/metadata/concepts_and_drugs/Concept.xml:4880]: @slubwama duplicate uuid '96370f02-aab1-4ec2-a42e-df73f63f12d4'

File: api/src/main/resources/metadata/concepts_and_drugs/Concept_Name.xml [hunk 2]
```diff
@@ -6796,5 +6796,7 @@
...
     <concept_name  concept_name_id="107467"  concept_id="159928"  name="school"  locale="en"  locale_preferred="0"  creator="1"  date_created="2010-11-29 01:42:21"  concept_name_type="FULLY_SPECIFIED"  voided="1"  voided_by="1"  date_voided="2024-05-28 12:30:19"  void_reason="Deleted due to change of name"  uuid="107467BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"  date_changed="2024-05-28 12:30:19"  changed_by="1"/>
     <concept_name  concept_name_id="187872"  concept_id="159928"  name="School, other institution, public administrative area"  locale="en"  locale_preferred="1"  creator="1"  date_created="2024-05-28 12:30:19"  concept_name_type="FULLY_SPECIFIED"  voided="0"   uuid="b87bf7b2-77cd-489c-b9df-dd22428b1984"  date_changed="2024-05-28 12:30:19"  changed_by="1"/>
-
+    <concept_name  concept_name_id="198843"  concept_id="186350"  name="ARV SPLIT DRUG SET"  locale="en"  locale_preferred="1"  creator="1"  ate_created="2024-06-18 12:40:41"  concept_name_type="FULLY_SPECIFIED"  voided="0"   uuid="b824100c-2d84-11ef-b0cb-e5066d1008f8"/>
```
Comment (Mwanje) [inline api/src/main/resources/metadata/concepts_and_drugs/Concept_Name.xml:6799]: @slubwama you missed a d for date_ceate, its throwing errors

File: api/src/main/resources/metadata/concepts_and_drugs/Concept_Name.xml [hunk 3]
```diff
@@ -6797,6 +6797,6 @@
...
-    <concept_name  concept_name_id="198844"  concept_id="175386"  name="Image attachment"  locale="en"  locale_preferred="1"  creator="2"  date_created="2024-04-14 15:40:17"  concept_name_type="FULLY_SPECIFIED"  voided="0"  uuid="552e757e-b072-4e68-a10d-b7db7efcd5ad" />
-    <concept_name  concept_name_id="198845"  concept_id="175385"  name="Default attachment"  locale="en"  locale_preferred="1"  creator="2"  date_created="2024-04-14 15:40:17"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="209ebadb-d3df-489a-bd10-7171e21c4869" />
+    <concept_name  concept_name_id="198844"  concept_id="186351"  name="Image attachment"  locale="en"  locale_preferred="1"  creator="2"  date_created="2024-04-14 15:40:17"  concept_name_type="FULLY_SPECIFIED"  voided="0"  uuid="552e757e-b072-4e68-a10d-b7db7efcd5ad" />
+    <concept_name  concept_name_id="198845"  concept_id="186352"  name="Default attachment"  locale="en"  locale_preferred="1"  creator="2"  date_created="2024-04-14 15:40:17"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="209ebadb-d3df-489a-bd10-7171e21c4869" />
```
Comment (semujju) [inline api/src/main/resources/metadata/concepts_and_drugs/Concept_Name.xml:6801]: @slubwama You also have ATT IMAGE ATTACHMENT and ATT DEFAULT ATTACHMENT. 
Then here you have Image attachment and Default attachment. 
Do you need to have all of them or it was some kind of repetition?

Comment (slubwama) [reply api/src/main/resources/metadata/concepts_and_drugs/Concept_Name.xml:6801]: When you follow  the concepts being create back to the attachment  module  you will notice  that. They only  get created  when they don't exist. What we are trying  to solve  here is the random  creation  of these concepts by creating  them first before the attachment  module  does.

Comment (semujju) [reply api/src/main/resources/metadata/concepts_and_drugs/Concept_Name.xml:6801]: This is noted @slubwama

**Review by semujju — APPROVED** [review]

**Pull request merged**